juriscraper/pacer/email.py

```diff
@@ -202,6 +204,40 @@ def _get_description_plain(self):
             return clean_string(description)
         return None
 
+    def _contains_attachments(self) -> bool:
+        """Determines if the html notification contains attached documents.
+
+        :returns: True if it contains otherwise False.
+        """
+
+        path = '//p[contains(., "The following document(s) are associated with this transaction:")]/following::'
+        document_nodes = self.tree.xpath(
+            f'{path}strong[contains(., "Document description:")]'
+        )
+        if len(document_nodes) <= 1:
+            return False
+        return True
+
+    def _contains_attachments_plain(self) -> bool:
+        """Determines if the plain/txt notification contains attached documents.
+
+        :returns: True if it contains otherwise False.
+        """
+
+        mail_body = self.tree.text_content()
+        regex = r"^.*?The following document\(s\) are associated with this transaction:(.*?)$"
+
+        find_attachments = re.findall(regex, mail_body, re.DOTALL)
+        associated_documents = 0
+        if find_attachments:
+            splitlines = find_attachments[0].splitlines()
+            for index_line in range(len(splitlines)):
+                if "Document description:" in splitlines[index_line]:
+                    associated_documents = +1
```

Comment:
Does this need to be so thorough? Why not just look for "The following documents...blah" and consider that enough?

Reply:
Yeah, the problem here is that even though a notification can contain `The following documents...` that doesn't mean that it has attachments because notifications that only have one document also have this phrase. That's why I had to count the number of documents with `Document description`